ui/views/mus/desktop_window_tree_host_mus.cc

```diff
@@ -318,6 +323,17 @@ bool DesktopWindowTreeHostMus::ShouldSendClientAreaToServer() const {
   return type == WIP::TYPE_WINDOW || type == WIP::TYPE_PANEL;
 }
 
+void DesktopWindowTreeHostMus::Relayout() {
```

Comment:
I will leave to you the option to 1) keep adding ::Relayout method or add it as a labda function within the #if block below, since it seems not needed to other general cases.

Reply:
As per our discussion, I said I was going to add a comment that this method should be shared with DesktopWindowTreeHostX11, but I just checked that I had already added a comment to the header file:

// TODO(msisov, tonikitoo): think how to share this with
  // DesktopWindowTreeHostX11. These two have pointers to
  // |native_widget_delegate_|.